ensembl_dbcopy/templates/admin/dbcopy/delete_selected_confirmation.html

```diff
@@ -0,0 +1,17 @@
+{% extends "admin/delete_selected_confirmation.html" %}
```

Comment:
Moved to ensembl_dbcopy. :pray:

Reply:
See previous message about project layout changes.